public/css/app.css

```diff
@@ -0,0 +1,80 @@
+@import url(https://fonts.googleapis.com/css2?family=Lato&family=Source+Serif+Pro&display=swap);
```

Comment:
Is this file automatically generated by Laravel mix? If so, I think we should avoid committing it (and `public/js/app.js`) into our repository (by adding it to `.gitignore`) and add a step to our build and deployment workflow to generate it.

Reply:
I was also wondering about this. Thanks for pointing it out.